src/api/viz.js

```diff
@@ -4,19 +4,21 @@ import * as schema from '../core/schema';
 import * as shaders from '../core/shaders';
 import { compileShader } from '../core/viz/shader-compiler';
 import { parseVizDefinition } from '../core/viz/parser';
-import Expression from '../core/viz/expressions/expression';
+import BaseExpression from '../core/viz/expressions/base';
```

Comment:
We are not in an expression file to it's clear for me to use `BaseExpression` because `Base` is not specific enough. It could be interpreted as a Base for viz or something else.